chainntnfs/txnotifier.go

```diff
@@ -1507,6 +1510,17 @@ func (n *TxNotifier) handleConfDetailsAtTip(confRequest ConfRequest,
 
 	// TODO(wilmer): cancel pending historical rescans if any?
 	confSet := n.confNotifications[confRequest]
+
+	// If we already have details for this request, we don't want to add it
+	// again since we have already dispatched notifications for it.
+	if confSet.details != nil {
+		Log.Infof("Not registering another notification for %v. "+
```

Comment:
Nit: would just reword this to `Ignoring address reuse for <script> at height <height>`.